lib/_tools.scss

```diff
@@ -429,9 +430,15 @@
 			}
 		} @else {
 			@for $i from 1 to $column {
-				.#{$gel-grid-namespace}#{$i}\/#{$column}#{$breakpoint-suffix} {
-					width: gel-columns($i / $column) !important;
-				}
+                $width: $i / $column;
```

Comment:
The indentation is mixed; my editor matched the most common. If we want it to be tabs, let's update the indentation in a separate PR.